Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```diff
@@ -180,9 +174,6 @@ FRFSQuote:
   types:
     FRFSQuoteType:
       enum: "SingleJourney, ReturnJourney, Pass, SpecialFareSingleJourney"
-    # For Bus it would be one route per leg, but for Metro it can be more than one route with transit stations for a single Journey leg.
-    FRFSRoutes:
-      enum: "Bus (Id Route), Metro [Id Route]"
```

Comment:
Not a good enum as even Bus can have multiple routes during Transits, so removed and changed the approach.